### DISCUSSION TITLE
Problem reproducing Protein-Ligand ST

### DISCUSSION THREAD

**Discussion by EdwardMendez95** [Valdes-Tresanco-MS/gmx_MMPBSA#41] [Q&A] [answered]
Greetings, I have a problem reproducing Protein-Ligand ST, using files you provide:

gmx_MMPBSA -O -i mmpbsa.in -cs com.tpr -ci index.ndx -cg 1 13 -ct com_traj.xtc -lm ligand.mol2 **-cr com.pdb** 

Using this, I obtain the error 

 File "/home/edward/anaconda3/envs/AmberTools20/lib/python3.9/site-packages/parmed/formats/pdb.py", line 703, in _end_model
    raise PDBError('Coordinate mismatch in model %d' % self._current_model_number)
PDBError: Coordinate mismatch in model 1
Exiting. All files have been retained.

When I use the same command without **-cr flag,** I do not have problem. I got this because I had the same problem with my system (using -cr flag in the command).

According to documentation, this flag is not mandatory but recommended (giving a .pdb as a guide to the scripts).

**Comment by marioernestovaldes** [comment, marked as answer]
Hi Edward and welcome! 

it is recommended to use the PDB file used to generate the topology in GROMACS, i.e. complex with the chain IDs and residue numbers you want to conserve and more important **No hydrogens** in it...

the flag -cr takes .pdb file as input and uses it as a reference file... that means chains IDs, residues numbers. etc from the reference structure will be used in the output files... as you can imagine this is useful in cases where you want to conserve the chain IDs or residues numbers in the output files, otherwise you can omit it... 

so, try working with com.pdb with no hydrogens and let us know

best!

**Reply by EdwardMendez95** [reply]
Oh I see, I will try. Thank you.

**Reply by Valdes-Tresanco-MS** [reply]
@EdwardMendez95 and @marioernestovaldes. I think the problem is in parmed. I have seen that it fails with PDB structures generated with GROMACS (editconf or trjconv) because it has the keyword MODEL. If this is the case, it is a bug that we can easily solve. I'm going to check this and also try to remove the hydrogens
